### PR TITLE
Update the Getting Started guide

### PR DESCRIPTION
**Pull request by obedrina**
Added the information about Bitbucket
@AleksandarCole could you please review when you have a chance? Thank you!

### CODE REVIEW

File: docs/guided-tour/getting-started.md [hunk 1]
```diff
@@ -30,14 +31,14 @@ To complete this tutorial, you will need:
...
+- A [GitHub][github] or [Bitbucket](http://bitbucket.com/) account
 
 ## Creating a Git repository
 
-Start by creating a new Git repository on GitHub. You may skip this section if you
+Start by creating a new Git repository on GitHub or Bitbucket. You may skip this section if you
 plan to use an existing repository.
 
-- Navigate to GitHub and [create a new repository][github-new].
+- Create a new repository on [Github][github-new] or [Bitbucket](https://bitbucket.org/new/)
```
Comment (markoa) [inline docs/guided-tour/getting-started.md:41]: follow the pattern of formatting links that's used throughout the page, where the name/URL pairs are defined at the bottom

File: docs/guided-tour/getting-started.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ block runs when it completes.
 **Blocks** can contain a single job or many parallel jobs. For example, your
 `Tests` block contains two parallel jobs.
 
-!!! Info: "Running parallel jobs simultaneously is possible under open source, free trial, or paid [plans][plans]."
+❗Info: "Running parallel jobs simultaneously is possible under open source, free trial, or paid [plans][plans]."
```
Comment (markoa) [inline docs/guided-tour/getting-started.md:123]: are you sure about this formatting change? it's special markup for an info box, it's not supposed to be an exclamation mark

**Review by markoa — CHANGES_REQUESTED** [review]
Overall good changes, it's some formatting that needs a revision. I took a look since it's been a while since you submitted and I created the original page.

**Review by obedrina — COMMENTED** [review]
Fixed the formatting as per the comments from Marko

**Comment by obedrina** [comment]
@markoa I updated the file as per the comments. Could you please take another look?
Thank you!

**Review by markoa — APPROVED** [review]
Good to go!

**Pull request merged**